external/storm-hdfs-blobstore/src/test/java/org/apache/storm/hdfs/blobstore/BlobStoreTest.java

```diff
@@ -535,6 +523,6 @@ public void testMultiple(BlobStore store)
             fail("Error the blobstore is of unknowntype");
         }
         assertStoreHasExactly(store, "test");
-        readAssertEquals(store, "test", 3);
+        readAssertEquals(store, "test", 4);
```

Comment:
This is a bugfix in the test. The test wasn't previously running, so this line hasn't been passing. Note that the test above writes 4 here https://github.com/apache/storm/pull/2990/files#diff-a48145dd5bbd83aced6dd85b1d85ec64L527. It is then reasonable to assert that the stored value is 4.